db/migrations/mysql/000082_threadmemberships_seenreplycount.up.sql

```diff
@@ -0,0 +1,26 @@
+SET @preparedStatement = (SELECT IF(
+    NOT EXISTS(
+        SELECT 1 FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'ThreadMemberships'
+        AND table_schema = DATABASE()
+        AND column_name = 'SeenReplyCount'
+    ),
+    'ALTER TABLE ThreadMemberships ADD COLUMN SeenReplyCount bigint(20) DEFAULT 0;',
+    'SELECT 1;'
+));
+
+PREPARE addColumnIfNotExists FROM @preparedStatement;
+EXECUTE addColumnIfNotExists;
+DEALLOCATE PREPARE addColumnIfNotExists;
+
+UPDATE
+	ThreadMemberships, Threads
+SET
+	SeenReplyCount = Threads.ReplyCount WHERE ThreadMemberships.LastViewed >= Threads.LastreplyAt AND ThreadMemberships.PostId = Threads.PostId;
```

Comment:
nit, not sure of the exact SQL style we follow, but can we pull `WHERE` down at least?
```suggestion
	SeenReplyCount = Threads.ReplyCount 
WHERE 
	ThreadMemberships.LastViewed >= Threads.LastReplyAt AND
	ThreadMemberships.PostId = Threads.PostId;
```

Reply:
Also, not `Threads.LastReplyAt` casing above.